apps/api/drizzle/schema.ts

```diff
@@ -0,0 +1,217 @@
+import { bigint, boolean, pgEnum, pgTable, text, timestamp, varchar } from 'drizzle-orm/pg-core';
+
+export const actionTypes = pgEnum('action_types', [
+	'CREATE',
+	'WITHDRAW',
+	'CLOSE',
+	'REQUEST_INSTITUTIONAL_REP',
+	'INSTITUTIONAL_REP_APPROVED',
+	'INSTITUTIONAL_REP_REJECTED',
+	'DAC_REVIEW_APPROVED',
+	'DAC_REVIEW_REJECTED',
+	'DAC_REVIEW_REVISIONS',
+	'REVOKE',
+]);
+export const agreementTypes = pgEnum('agreement_types', [
+	'dac_agreement_software_updates',
+	'dac_agreement_non_disclosure',
+	'dac_agreement_monitor_individual_access',
+	'dac_agreement_destroy_data',
+	'dac_agreement_familiarize_restrictions',
+	'dac_agreement_provide_it_policy',
+	'dac_agreement_notify_unauthorized_access',
+	'dac_agreement_certify_application',
+	'dac_agreement_read_and_agreed',
+]);
+export const applicationStates = pgEnum('application_states', [
+	'DRAFT',
+	'INSTITUTIONAL_REP_REVIEW',
+	'DAC_REVIEW',
+	'DAC_REVISIONS_REQUESTED',
+	'REJECTED',
+	'APPROVED',
+	'CLOSED',
+	'REVOKED',
+]);
+export const fileTypes = pgEnum('file_types', ['SIGNED_APPLICATION', 'ETHICS_LETTER']);
+
+export const actions = pgTable('actions', {
+	// You can use { mode: "bigint" } if numbers are exceeding js number limitations
+	id: bigint({ mode: 'number' }).primaryKey().generatedAlwaysAsIdentity({
+		name: 'actions_id_seq',
+		startWith: 1,
+		increment: 1,
+		minValue: 1,
+		maxValue: 9223372036854775807,
+		cache: 1,
+	}),
+	// You can use { mode: "bigint" } if numbers are exceeding js number limitations
+	applicationId: bigint('application_id', { mode: 'number' }).notNull(),
+	createdAt: timestamp('created_at', { mode: 'string' }).defaultNow().notNull(),
+	userId: varchar('user_id', { length: 100 }).notNull(),
+	action: actionTypes().notNull(),
+	// You can use { mode: "bigint" } if numbers are exceeding js number limitations
+	revisionsRequestId: bigint('revisions_request_id', { mode: 'number' }),
+	stateBefore: varchar('state_before', { length: 255 }).notNull(),
+	stateAfter: varchar('state_after', { length: 255 }).notNull(),
+});
+
+export const agreements = pgTable('agreements', {
+	// You can use { mode: "bigint" } if numbers are exceeding js number limitations
+	id: bigint({ mode: 'number' }).primaryKey().generatedAlwaysAsIdentity({
+		name: 'agreements_id_seq',
+		startWith: 1,
+		increment: 1,
+		minValue: 1,
+		maxValue: 9223372036854775807,
```

Comment:
This was generated using `drizzle-kit pull` so it's the auto-generated upper limit for a BigInt

Reply:
Leo and I chatted about this -- there is a disconnect between the DBML schema input (expects an object) and the Drizzle config schema definition (string glob path to schemas folder './schemas/**')

This should be refactored so that both point to the same schema definition, my only hesitation is to limit the scope of this PR. I will update this and the test files in the next branch.